content/create-an-addon/react/fr/getting-started.md

```diff
@@ -21,15 +21,19 @@ Pour commencer, cliquez sur le bouton **Use this template** sur le [dépôt de l
 Ensuite, clonez le dépôt et installez les dépendances.
 
 ```bash
-npm install
-# Utiliser la commande suivante pour npm 7
-npm i --legacy-peer-deps
+yarn
+```
+
+L'Addon Kit utilise TypeScript par défaut. Toutefois, nous utiliserons la commande eject pour convertir le code standard en JavaScript dans le cadre de ce tutoriel.
```

Comment:
@Dschungelabenteuer added a note to account for some changes coming to the addon kit. Does this make sense grammatically?

Reply:
Sorry for the delay I was taking some time off! It definitely makes sense gramatically. I'd replace "code standard" with something like "code de base" though (this is quite a hard-to-translate translation unit to be honest but this would fit right in) because "code standard" implies and makes the end reader think of... well... an official standardized way of writing code. I'd push things a bit further by stating "Toutefois, nous utiliserons la commande eject pour convertir le code de base en simple JavaScript". Here "simple" would mean "pure". Any thoughts @frassinier ?

Reply:
> L'Addon Kit est écrit en TypeScript par défaut mais, par soucis de simplification pour ce tutoriel, nous utiliserons la commande eject pour avoir du simple JavaScript.

Maybe more easy to understand?